lib/debug/frame_filter.rb

```diff
@@ -0,0 +1,10 @@
+module DEBUGGER__
+  def self.register_frame_filter(&block)
+    @frame_filters ||= []
```

Comment:
I imagine there could be multiple debugger extensions in the future, so having multiple filters would be possible.